### PR TITLE
UI issues in Calculated Icons

### PR DESCRIPTION
**Pull request by sravfeyn**
http://manage.dimagi.com/default.asp?233227#edit_0_233227 @nickpell

### CODE REVIEW

File: corehq/apps/hqwebapp/static/hqwebapp/js/key-value-mapping.js [hunk 1]
```diff
@@ -98,6 +98,23 @@ function MapList(o) {
     self.multimedia = o.multimedia;
     self.property_name = o.property_name;
 
+    self.labels = ko.computed(function() {
+        if (this.values_are_icons()) {
+            return {
+                placeholder: 'Calculation',
```
Comment (orangejenny) [inline corehq/apps/hqwebapp/static/hqwebapp/js/key-value-mapping.js]: translations please (`gettext`)

**Review by orangejenny — CHANGES_REQUESTED** [review]

File: corehq/apps/hqwebapp/static/hqwebapp/js/key-value-mapping.js (hunk 1 above)
Comment (sravfeyn) [inline corehq/apps/hqwebapp/static/hqwebapp/js/key-value-mapping.js]: Oh yeah, I was gonna add it. Thanks!

**Comment by dimagimon** [comment]
There are 6 errors:

* corehq/apps/hqwebapp/static/hqwebapp/js/key-value-mapping.js, line 104 - 'django' is not defined. (no-undef)
* corehq/apps/hqwebapp/static/hqwebapp/js/key-value-mapping.js, line 105 - 'django' is not defined. (no-undef)
* corehq/apps/hqwebapp/static/hqwebapp/js/key-value-mapping.js, line 106 - 'django' is not defined. (no-undef)
* corehq/apps/hqwebapp/static/hqwebapp/js/key-value-mapping.js, line 111 - 'django' is not defined. (no-undef)
* corehq/apps/hqwebapp/static/hqwebapp/js/key-value-mapping.js, line 112 - 'django' is not defined. (no-undef)
* corehq/apps/hqwebapp/static/hqwebapp/js/key-value-mapping.js, line 113 - 'django' is not defined. (no-undef)

**Review by orangejenny — APPROVED** [review]

**Comment by orangejenny** [comment]
Looks good, just holding off on merge because of label.

**Comment by sravfeyn** [comment]
@orangejenny I've removed the label, jenny.

**Pull request merged**